test_scripts/API/AutoCompleteList/006_ATF_SetGlobalProperties_P_AutoCompleteList_ValueSpaceAfter.lua

```diff
@@ -0,0 +1,81 @@
+----------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL  must transfer request in case of valid "autoCompleteList" param
```

Comment:
@Aleksandriys, please correct summary of requirement to be exactly the same as in Jira:
"[SetGlobalProperties] SDL must transfer request to HMI in case of valid "autoCompleteList" param"
 and not: 
"[SetGlobalProperties] **Conditions for** SDL  must transfer request in case of valid "autoCompleteList" param"